README.md

```diff
@@ -5,6 +5,8 @@ PROCESS is the reactor systems code at the [UK Atomic Energy Authority](https://
 
 PROCESS was originally a Fortran code, but is currently a mixture of Python and Python-wrapped Fortran; the eventual aim is to have an entirely Python code base. In order to use PROCESS, the Fortran must be compiled and a Python-Fortran interface generated for the Python to import. Once built, it can be installed and run as a Python package.
 
+![BlenderProcessImage](blenderimages/initial_collage.png)
```

Comment:
Minor: could this go into `images` instead or `blenderimages`? We might well have more non-blender images in future.